internal/learn/the-community/qanda.md

```diff
@@ -0,0 +1,98 @@
+
+All native tokens are standalone tokens in the ledger that are brought into existence by foundries and can be held or transferred from any account in the ledger. Native tokens are injected into the IOTA ledger by users. Therefore, they are also referred to as user-defined tokens. They consume valuable resources of the nodes that maintain the network, primarily storage space. As a result, each account holding native tokens must post a deposit in IOTA coins to compensate for the excessive resource consumption.
+
+Or in other words, storage space in the IOTA ledger costs IOTA tokens (deposit). When the storage space is no longer needed, the deposited tokens can be transferred or sold.
+
+**Micro payments**
+The absence of miners makes IOTA transactions completely charge-free transactions. Send 1 MIOTA = receive 1 MIOTA . This enables real micro payments for the machine-to-machine economy so that payment for small quantities or consumption-based payments can be implemented.
+
+No micro payments without the IOTA token.
+
+**Accounting**
+In discussions, there was always the suggestion that you could combine individual micro-transactions and then pay monthly, for example as with a subscription. If one would try this as a company it would require the creation of an accounting system that is able to handle millions of data-Tx.
+
+These need to be…
+
+1. evaluated (because they can all have completely different prices).
+2. assigned (thousands of data tx would have to be assigned to the respective user).
+3. calculated (for each user a final invoice would have to be made and possibly be charged with their data-tex if you are a user yourself).
+4. check (that the payments have actually been made).
+
+That would result in…
+
+1. an immense staff body
+2. a high risk of errors
+3. a high demand for IT and office equipment for the personnel in question
+4. thus high and hardly calculable costs.
+
+Conclusion: Completely unnecessary effort of resources and personnel, which the protocol provides free of charge and fully automated with the help of IOTA Smart Contracts, IOTA streams etc.
+
+Btw.: All companies have to document and tax value transactions. So all data transactions have to be recorded in blockchains for accounting purposes. With IOTA, not only the fees for data transactions are eliminated but also the associated accounting including Tx storage (up to 10 years) for the majority of all transactions.
+
+**Chain of events**
+Only with a system in which data transactions can be linked directly to value transactions the complete event chain of the data trade can be correctly traced. The IOTA token reflects the (potential) value generated by the data transaction. Therefore, it would make no sense at all if only the IOTA data transactions were used but the payment of this data takes place outside the IOTA protocol.
```

Comment:
```suggestion
Only with a system in which data transactions can be linked directly to value transactions can the complete event chain of the data trade be correctly traced. The IOTA token reflects the (potential) value generated by the data transaction. Therefore, it would make no sense if only the IOTA data transactions were used, but the payment of this data takes place outside the IOTA protocol.
```